content/docs/start/experiments-trail/experiments-1.md

```diff
@@ -0,0 +1,334 @@
+
+<details>
+
+### Installing and Configuring the Project
+
+These commands are run in the [`get-started-experiments`][gse] project. You can
+run the commands in this document after cloning the repository and installing
+the requirements.
+
+### Clone the project and create virtual environment
+
+Please clone the project and create a virtual environment.
+
+> We create a virtual environment to keep the libraries we use isolated from the
+> rest of your system. This prevents version conflicts.
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments -b source-code
+$ cd get-started-experiments
+$ virtualenv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+### Get the data set
+
+The repository you cloned doesn't contain the dataset. In order to get
+`fashion-mnist.tar.gz` from `dataset-registry`, we use `dvc get` to download the
+missing data files. `dvc get` is similar to `wget` but works with Git+DVC
+repositories to download binary files.
+
+```dvc
+$ dvc get https://github.com/iterative/dataset-registry \
+          fashion-mnist/images.tar.gz -o data/images.tar.gz
+```
+
+Then we extract this file that contains labeled images.
+
+```dvc
+$ tar -xvzf data/images.tar.gz --directory data/
+```
+
+### Specify the experiment
+
+We first initialize DVC inside the project to create an experiment.
+
+```dvc
+$ dvc init
+```
+
+Then we add the dataset to the project:
+
+```dvc
+$ dvc add data/images
+```
+
+This creates a `data/images.dvc` file that contains all the relevant metadata of
+the directory. You can add and commit `data/images.dvc` file, and changes in
+`.gitignore` that hides the `data/images/` directory from Git.
+
+```dvc
+$ git add data/.gitignore data/images.dvc
+$ git commit -m "Dataset added"
+```
+
+DVC experiments are run by specifying their commands, outputs, parameters and
+dependencies. We specify an experiment command by `dvc stage add`.
+
+```dvc
+$ dvc stage add -n train \
+                -p model.conv_units \
+                -p train.epochs \
+                -d data/images \
+                -m metrics.json \
+                python3 src/train.py
+```
+
+The command tells DVC to create an experiment named `train`, and that for any
+change in `data/images/`, `model.conv_units` or `train.epochs`, we run an
+experiment using `src/train.py` that produces a new `metrics.json` file.
+
+DVC creates `dvc.yaml` file and modifies `.gitignore` for Git to ignore certain
+artifacts.
+
+```dvc
+$ git add dvc.yaml .gitignore
+$ git commit -m "added data and the experiment"
+```
+
+DVC is ready to run the experiments now!
+
+</details>
+
+## Running the experiment with default parameters
+
+The purpose of `dvc exp` subcommands is to run the pipeline for ephemeral
+experiments. By _ephemeral_ we mean the experiments can be run without
+committing parameter and dependency changes to Git. Instead the artifacts
+produced for each experiment are tracked by DVC and persisted on demand.
```

Comment:
`ephemeral experiments`: at some point we decided not to use that term. All experiments are the same, but some you can chose to make persistent as Git commits.

> For some reason the term has lingered internally e.g. Dmitry uses it.

Reply:
:) I see. I need some word to describe _experiments not yet persisted and may be thrown out._ I agree that _ephemeral_ looks like we are working in void and experiments are like clouds, but what can it be? I added the explanation because of this. 

_Impermanent_ feels experiments are about to collapse. 

All experiments are not the same, we breed them and select some of them as our next-generation livestock.

Reply:
> I need some word to describe experiments not yet persisted and may be thrown out.

That's my point though, ideally you don't need to make that distinction. The general concept of "experiment" already implies that it's ephemeral or disposable.

So you can just say something like "we're making experiments here blah blah..." And later in the **Persisting** section then you go into explaining what persistent experiments are (Git commits or branches, i.e. arbitrary project versions).